components/server/ee/src/workspace/workspace-factory.ts

```diff
@@ -63,61 +62,28 @@ export class WorkspaceFactoryEE extends WorkspaceFactory {
             const imageSource = await this.imageSourceProvider.getImageSource(ctx, user, context.actual, config);
 
             // Walk back the commit history to find suitable parent prebuild to start an incremental prebuild on.
+            // TODO this doesn't work for multi-repo projects. We should fetch the last few available prebuilds and iterate over them instead.
```

Comment:
nit: please adapt to the TODO comment convention (https://www.notion.so/gitpod/How-we-develop-e8ce7e562478458a9223eb9b797415b2#86e669f8e7df4f1289c9bede0855b5f9)